canopeum_frontend/src/components/analytics/site-modal/SiteModal.tsx

```diff
@@ -291,7 +292,7 @@ const SiteModal = ({ open, handleClose, siteId }: Props) => {
       <DialogActions>
         <button
           className='btn btn-outline-primary'
-          onClick={() => handleClose()}
+          onClick={() => handleClose('cancel')}
```

Comment:
`undefined` was kindof implied as `'cancel'`. But explicit is better than implicit right? Maybe we should make the `reason` param required ?

Reply:
![image](https://github.com/user-attachments/assets/eddfa49f-6abc-4b48-94ea-fef96b02d9e3)
I resolved it :(